src/mgr/ActivePyModules.cc

```diff
@@ -38,6 +38,8 @@
 #include "PyModuleRegistry.h"
 #include "PyUtil.h"
 
+#include "common/perf_counters_key.h"
```

Comment:
move the include line to its 'correct' place alphabetically.

Reply:
done!